rero_mef/agents/mef/api.py

```diff
@@ -183,6 +183,33 @@ def create_deleted(cls, record, dbcommit=False, reindex=False):
         }
         return cls.create(data=data, dbcommit=dbcommit, reindex=reindex)
 
+    @classmethod
+    def get_latest(cls, pid_type, pid):
+        """Get latest Mef record for pid_type and pid.
+
+        :param pid_type: pid type to use.
+        :param pid: pid to use..
+        :returns: latest record.
+        """
+        search = AgentMefSearch().filter({'term': {f'{pid_type}.pid': pid}})
```

Comment:
Is moustache are necessary ? `AgentMefSearch().filter('term', f'{pid_type}.pid'=pid)` ?

Reply:
I think so.